### PR TITLE
Add keyboard navigation to Tree Grid overview topic

### LINKED ISSUE #804: For treeGrid there should be a topic for keyBoard navigation
There should be an information for keyBoard navigation:
- Row can be collapsed with Alt+Arrow Left key
- Row can be expanded with Alt+ Arrow Right key

### PR DESCRIPTION
**Pull request by tachojelev**
Closes #804

### CODE REVIEW

File: en/components/treegrid/tree_grid.md [hunk 1]
```diff
@@ -236,6 +236,38 @@ The indentation of the **tree cells** persists across other tree grid features l
...
+ - `Arrow Left` - navigates one cell left (no wrapping between lines)
+ - `Arrow Right` - navigates one cell right (no wrapping between lines)
+ - `Ctrl + Arrow Up` - navigates to the first cell in the current column;
+ - `Ctrl + Arrow Down` - navigates to the last cell in the current column;
+ - `Ctrl + Arrow Left` -  moves to leftmost cell in row
+ - `Home` - moves to leftmost cell in row
+ - `Ctrl + Home` - moves to leftmost cell in row
+ - `Ctrl + Arrow Right` - moves to rightmost cell in row
+ - `End` - moves to rightmost cell in row
+ - `Ctrl + End` - moves to rightmost cell in row
```
Comment (mtsvyatkova) [inline en/components/treegrid/tree_grid.md]: Probably using one of the bellow descriptions is more accurate:
`Ctrl + End` - moves to bottom right cell in the grid
or
`Ctrl + End` - moves to rightmost cell of the last row

File: en/components/treegrid/tree_grid.md [hunk 2]
```diff
@@ -236,6 +236,38 @@ The indentation of the **tree cells** persists across other tree grid features l
...
+
+ - `Arrow Up` - navigates one cell up (no wrapping)
+ - `Arrow Down` - navigates one cell down (no wrapping)
+ - `Arrow Left` - navigates one cell left (no wrapping between lines)
+ - `Arrow Right` - navigates one cell right (no wrapping between lines)
+ - `Ctrl + Arrow Up` - navigates to the first cell in the current column;
+ - `Ctrl + Arrow Down` - navigates to the last cell in the current column;
+ - `Ctrl + Arrow Left` -  moves to leftmost cell in row
+ - `Home` - moves to leftmost cell in row
+ - `Ctrl + Home` - moves to leftmost cell in row
```
Comment (mtsvyatkova) [inline en/components/treegrid/tree_grid.md]: Probably using one of the bellow descriptions is more accurate:
`Ctrl + Home` - moves to top left cell in the grid
or
`Ctrl + Home` - moves to leftmost cell of the first row

File: en/components/treegrid/tree_grid.md [hunk 3]
```diff
@@ -236,6 +236,38 @@ The indentation of the **tree cells** persists across other tree grid features l
 
 <div class="divider--half"></div>
 
+### Keyboard navigation
+
+Keyboard navigation is available by default in the Tree Grid and aims at covering as many as possible features and scenarios for the end user. When you focus a specific cell and press one of the following key combinations, the described behaviour is performed:
+
+ - `Arrow Up` - navigates one cell up (no wrapping)
```
Comment (mtsvyatkova) [inline en/components/treegrid/tree_grid.md]: May be we should be using semicolons or dots at the end of all rows about key combinations

File: en/components/treegrid/tree_grid.md (hunk 1 above)
Comment (DiyanDimitrov) [reply en/components/treegrid/tree_grid.md]: We should change that for Grid and Hierarchical Grid too.

File: en/components/treegrid/tree_grid.md (hunk 2 above)
Comment (DiyanDimitrov) [reply en/components/treegrid/tree_grid.md]: We should change that for Grid and Hierarchical Grid too.

**Pull request merged**